### PR TITLE
Add digitalocean extension

### PR DESCRIPTION
**Pull request by ShukantPal**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/digitalocean/package.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  "author": "shukantpal",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-container-registry",
+      "title": "Search Container Registry",
+      "subtitle": "DigitalOcean",
```
Comment (erics118) [inline extensions/digitalocean/package.json:16]: This subtitle is actually redundant. If there is no title field, it already defaults to the extension title.

Comment (pernielsentikaer) [reply extensions/digitalocean/package.json:16]: That's only happening in development mode. I'm pretty sure that in production, it actually makes sense to have it like this.

**Comment by ShukantPal** [comment]
I’m curious what are the next steps to get this published.

On Oct 23, 2023, at 5:22 AM, Per Nielsen Tikær ***@***.***> wrote:



@pernielsentikaer commented on this pull request.

________________________________

In extensions/digitalocean/package.json<https://github.com/raycast/extensions/pull/8909#discussion_r1368572595>:

> +{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "digitalocean",
+  "title": "DigitalOcean",
+  "description": "Quickly find resources in your DigitalOcean team",
+  "icon": "command-icon.png",
+  "author": "shukantpal",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-container-registry",
+      "title": "Search Container Registry",
+      "subtitle": "DigitalOcean",


That's only happening in development mode. I'm pretty sure that in production, it actually makes sense to have it like this.

**Comment by pernielsentikaer** [comment]
Could you help with a demo screencast of the extension since I don't have Digital Ocean?

**Comment by ShukantPal** [comment]
I have created a video of my usage of the extension!

https://github.com/raycast/extensions/assets/22450567/53dd0bab-d934-4018-9f2b-ccd4f4dd376c

**Comment by ShukantPal** [comment]
@pernielsentikaer Here you go!

https://github.com/raycast/extensions/assets/22450567/133d790f-844d-45cb-9321-a6fee1d0ee35

**Comment by ShukantPal** [comment]
@pernielsentikaer Let me know if there's something you need to me to clarify or record!

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Awesome, thanks for the video 🫶 Could you add the last missing icons so all action panel items has one to keep the UI consistent 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by pernielsentikaer** [comment]
Could you help with some [metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) @ShukantPal  🙂

**Pull request merged**